spec/solver_spec.rb

```diff
@@ -0,0 +1,50 @@
+require_relative '../solver'
+
+describe Solver do
```

Comment:
- - Requirement screenshot:
![Screenshot from 2023-11-16 09-29-30](https://github.com/siddghosh108/TDD-Project/assets/107632780/33733951-b3dd-44ac-9a00-350e9480dc3a)

- [ ] Observing your commit history, it appears that the `Test-Driven Development (TDD)` approach might not have been strictly followed. The typical TDD cycle involves writing tests first, `committing, pushing`, and then proceeding with the implementation of the class and method. This practice ensures a clear roadmap for development and enhances the overall robustness of the codebase. Consider adhering to this cycle for future contributions to reinforce code quality and adherence to specifications. :smile: :small_airplane:

Reply:
I do agree and will follow your sugesstions in future